example/animation.rb

```diff
@@ -22,13 +22,14 @@ def out.tty?; true; end
   lines.each do |line|
     $stdout.print "\r#{line}"
   end
+  $stdout.print "\e[0J"
```

Comment:
@kojix2 Why is this necessary?

Reply:
It is not necessary, but it will ensure that there are no more writes.

Reply:
For example, if you hit the keyboard repeatedly during execution, the characters will be erased.

Reply:
I got it. Thank you!